crates/factor-outbound-mysql/src/host.rs

```diff
@@ -38,8 +40,15 @@ impl<C: Client> v2::Host for InstanceState<C> {}
 
 #[async_trait]
 impl<C: Client> v2::HostConnection for InstanceState<C> {
-    #[instrument(name = "spin_outbound_mysql.open_connection", skip(self), err(level = Level::INFO), fields(otel.kind = "client", db.system = "mysql"))]
+    #[instrument(name = "spin_outbound_mysql.open", skip(self, address), err(level = Level::INFO), fields(otel.kind = "client", db.system = "mysql", db.address = Empty, server.port = Empty, db.namespace = Empty))]
     async fn open(&mut self, address: String) -> Result<Resource<Connection>, v2::Error> {
+        if let Ok(url) = Url::parse(&address) {
```

Comment:
I wish we could capture this logging functionality in one place instead of copying it. In particular I'm most interested in capturing a comment about how we specifically _don't_ want to log the full address due to it potentially containing sensitive data. 

How would this look if we moved this to a function in `spin-factor-outbound-networking`?

Reply:
I've addressed this (pun intended)